man/coreos-installer-install.8

```diff
@@ -56,6 +56,11 @@ Override the Ignition platform ID
 
 Install a system that will run on the specified cloud or virtualization platform, such as "vmware".
 .TP
+\fB\-\-console\fR=\fIspec\fR
+Kernel and bootloader console
+
+Set the kernel and bootloader console, using the same syntax as the parameter to the "console=" kernel argument.
```

Comment:
Silly doubt: is the user-specified value expected to include the `console=` part or not (I think the answer is "no")? Should we try to handle that explicitly?

Reply:
It isn't.  I've added a specific error message for this case.